test/circleci/rollcage/test_core.clj

```diff
@@ -202,7 +202,7 @@
             {err :err skipped :skipped {uuid :uuid} :result} (client/warning r e)]
         (is (zero? err))
         (is (true? skipped))
-        (is (UUID/fromString uuid))))))
+        (is (and uuid (UUID/fromString uuid)))))))
```

Comment:
Why would uuid by nil?

Reply:
This is unrelated to our change. When something goes wrong with execution, test throws an exception trying to parse nil UUID rather than failing an assertion. This is not really necessary for the purpose of this PR, I can back it out.